src/localization/locales/lang_hu.lua

```diff
@@ -29,4 +29,7 @@ _Literals = {
     CONNECTION_CHARACTER_SELECTION_BUTTON = "Játék vele %s",
 
     CONNECTING_CREATING_CHARACTER = "Karakter létrehozása...",
+
+    KICK_DEFAULT_MESSAGE = "Ki lettél kikerülve a szerverről.",
+    BAN_DEFAULT_MESSAGE = "A szerver tiltották a kapcsolatod.",
```

Comment:
```suggestion
    KICK_DEFAULT_MESSAGE = "Ki lettél dobva a szerverről.",
    BAN_DEFAULT_MESSAGE = "Ki lettél tiltva a szerverről.",
```

Reply:
copilot lol, thanks
that's correct ?     UNBAN_DEFAULT_MESSAGE = "A szerver engedélyezte a kapcsolatod.",

Reply:
the correct is 
```
UNBAN_DEFAULT_MESSAGE = "A kitiltásod feloldva.",
```

Reply:
ok thx